_commands/tools/wget.md

```diff
@@ -3,22 +3,48 @@
 
 wget
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+wget stands for "web get". It is a command-line utility which downloads files over a network.
 
+The main advantages of wget are it is free for non-interactive download, meaning that it can work in the background and while the user is not logged on, and it supports HTTP, HTTPS, and FTP protocols, as well as retrieval through HTTP proxies.
 <!-- minimal example -->
+
 ~~~ bash
-*short* example of command usage and output
+$ wget http://example.com/file.iso
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+\-\-mirror #turn on mirroring
+
+\-\-adjust-extension #add proper extension to downloaded files
+
+\-\-page-requisites #download css and other required files
+
+\-\-execute robots=off #ignore robots.txt
+
+\-\-wait=30 #wait between fetches and requests
+
+\-\-convert-links #convert links for offline viewing
+
+\-\-user-agent=C4CS #fake the User Agent
 
 #### Example command
 
-##### Break it down
+~~~ bash
+$ wget ‐‐output-document=filename.html example.com
```

Comment:
these are two different example commands, they should have their own subheading, code block, and explanation